docs/openj9_signals.md

```diff
@@ -178,6 +182,10 @@ command line syntax to use with the compiler, where available:
 | Windows                | `jsig.dll`     | Link the DLL with the application that creates or embeds a VM |
 | AIX                    | `libjsig.so`   | `cc_r [-q64] <other_compile/link_parameter> -L<java_install_dir> -ljsig -L<java_install_dir>/lib/j9vm -ljvm <java_application>.c` |
 
+With the `-XX:-HandleSIGUSR2` option, the VM signal handler is not installed even on receiving the `SIGUSR2` signal, which targets the VM. Therefore, even if the signal handler chaining library is not loaded, the VM handler does not replace the application signal handler, if available, when handling the `SIGUSR2` signal.
```

Comment:
The signal handler is installed on VM startup. It is not installed upon receiving the `SIGUSR2` signal. 
```suggestion
With the `-XX:-HandleSIGUSR2` option, the VM signal handler is not installed. In the absence of the signal handler chaining library, the application signal handler, if available, is not overridden by the VM signal handler for handling the `SIGUSR2` signal.
```